google/resource_compute_security_policy.go

```diff
@@ -53,6 +53,14 @@ func resourceComputeSecurityPolicy() *schema.Resource {
 				Description: `The project in which the resource belongs. If it is not provided, the provider project is used.`,
 			},
 
+			"type": {
+				Type:        schema.TypeString,
+				Optional:    true,
+				ForceNew:    true,
+				Default:     "CLOUD_ARMOR",
```

Comment:
It may be best to not specify a default value for this but instead just not send it the value is not set.

Reply:
Removed default